net-im/skypeforlinux/skypeforlinux-9999.ebuild

```diff
@@ -9,7 +9,7 @@ inherit eutils rpm
 
 DESCRIPTION="P2P Internet Telephony (VoiceIP) client"
 HOMEPAGE="http://www.skype.com/"
-SRC_URI="https://repo.skype.com/rpm/stable/skypeforlinux_${PV}-1.x86_64.rpm"
+SRC_URI="https://repo.skype.com/latest/skypeforlinux-64-alpha.rpm"
```

Comment:
I'll take a peek and try to get a version number.

Upstream doesn't supply version numbers though, so I'm worried that sticking a version on the ebuild will cause misleading installs.  AFAIK there isn't a way to tell if upstream version number's changed.